tools/cdb2sql/cdb2sql.cpp

```diff
@@ -175,8 +176,31 @@ const char *words[] = {
   "SELECT", "SELECTV", "SET",
   "TRUNCATE",
   "UPDATE",
-  "WITH", NULL, }; // must be terminated by NULL
+  "WITH", NULL,  // must be terminated by NULL
+};
+
+
+const char *char_atglyph_words[] = {
+    "desc", "ls", "send", NULL, // must be terminated by NULL
+};
+
 
+char *char_atglyph_generator (const char *text, int state)
+{
+    static int list_index, len;
+    const char *name;
+    if (!state) { //if state is 0 get the length of text
+        list_index = 0;
+        len = strlen (text);
+    }
+    while ((name = char_atglyph_words[list_index]) != NULL) {
+        list_index++;
+        if (len == 0 || strncasecmp (name, text, len) == 0) {
+            return strdup (name);
+        }
+    }
+    return ((char *) NULL); // If no names matched, then return NULL.
```

Comment:
superfluous cast?

Reply:
This is a separate executable, but ok sure.